yolo-object-detection.html

```diff
@@ -3,7 +3,8 @@
         category: 'function',
         color: '#a6bbcf',
         defaults: {
-            detectionServerPort: {value:"8885"}
+            detectionServerPort: {value:"8885"},
```

Comment:
```suggestion
            detectionServerPort: {value:"8888"},
```

Default port is 8888 last time I looked?

Reply:
Thanks Ben.  I think it was originally 8888, but got changed to 8885?  Just checked references in the code and saw it is set by default to 8888 in the Python and .js files, but set to 8885 in the .html.  I'll set it to 8888 in the .html to be consistent across all three.